packages/core/web-reporter-ui/src/components/Chart.tsx

```diff
@@ -120,86 +132,64 @@ export const Chart = ({
 
   const videoEnabled = useContext(VideoEnabledContext);
 
-  const options = useMemo<ApexOptions>(
```

Comment:
ah this `useMemo` is actually useful, so that `options` doesn't change when `series` change

When using `flashlight measure` the measures are dynamic, `series` change but `options` should not. If it does the graph becomes really slow